lib/src/node/value/calculation.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2023 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:collection/collection.dart';
+import 'package:node_interop/js.dart';
+import 'package:sass/src/node/immutable.dart';
+import 'package:sass/src/node/utils.dart';
+
+import '../../value.dart';
+import '../reflection.dart';
+
+/// Check that [arg] is a valid argument to a calculation function.
+void assertCalculationValue(Object arg, {bool checkUnquoted = true}) {
```

Comment:
I'm updating the spec so that `CalculationOperation` will check for unquoted strings, which means you can get rid of `checkUnquoted` here _and_ the `CalculationOperation` check below.